backend/src/routes.py

```diff
@@ -13,3 +16,33 @@ def apitest():
     }
 
     return res
+
+
+def load_json():
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    questions_file_path = os.path.join(
+        dir_path, 'static', 'questions.json')
+    try:
+        with open(questions_file_path, 'r') as file:
```

Comment:
Specify encoding for the file read, pylint is crying about this.

```suggestion
        with open(questions_file_path, 'r', encoding='utf-8') as file:
```